quartz_solar_forecast/data.py

```diff
@@ -9,9 +9,13 @@
 import xarray as xr
 
 from quartz_solar_forecast.pydantic_models import PVSite
+from inverters.enphase import get_enphase_data # Added import for get_enphase_data from /inverters/enphase.py
 
 ssl._create_default_https_context = ssl._create_unverified_context
 
+# User needs to add their Enphase API details
+ENPHASE_API_KEY = 'user_enphase_api_key'
```

Comment:
could we make this envionrment variables?

Reply:
> could we make this envionrment variables?

Sure, will make the required change and add a .env.example file

Reply:
> could we make this envionrment variables?

Should i add a .env file in /Open-Source-Quartz-Solar-Forecast/quartz_solar_forecast
or simply in /Open-Source-Quartz-Solar-Forecast

Reply:
> could we make this envionrment variables?

I have added the changes in my PR for environment variables, please review it. Thank you